libvirt/tests/src/virtual_network/attach_detach_device/attach_detach_ovs_interface.py

```
@@ -0,0 +1,155 @@
+
+def setup_ovs_bridge(bridge_name, bridge_ip, bridge_netmask):
+    """
+    Create OVS bridge if it doesn't exist and assign an IP address to it.
+
+    :param bridge_name: name of the OVS bridge
+    :param bridge_ip: IP address to assign to the bridge
+    :param bridge_netmask: prefix length (e.g. "24")
+    """
+    if not utils_net.ovs_br_exists(bridge_name):
+        utils_net.add_ovs_bridge(bridge_name)
+    process.run("ip link set %s up" % bridge_name, shell=True)
```

Ruff (0.15.20)

[error] Function call with `shell=True` parameter identified, security issue

(S604)